src/ui/components/ShowMoreCard/ShowMoreCard.scss

```diff
@@ -27,3 +28,21 @@
     max-height: none;
   }
 }
+
+.AddonDescription {
+  .ShowMoreCard-contents {
+    @include respond-to(large) {
+      &::after {
+        height: 0;
+      }
+
+      max-height: none;
+    }
+  }
+
+  footer.Card-footer-link {
```

Comment:
Is this only ever used for "show more" purposes? Does it actually need `footer` for specificity?

Reply:
You're correct, it doesn't need `footer`. I will remove it.